generator-mediator-js/app/templates/packagingTemplate/targets/trusty/etc/init/_mediator.conf

```diff
@@ -0,0 +1,20 @@
+# <%= mediatorName %>
```

Comment:
It happens [here](https://github.com/jembi/openhim-mediator-yeoman-generators/blob/update-generator-js/generator-mediator-js/app/index.js#L93), but I see I missed the .conf, so thanks for pointing it out!